### PR TITLE
Update hover designs

### PR DESCRIPTION
**Pull request by fonurr**
* Limit number of Zeplin components shown on hover
* Update Open In Zeplin links on project, styleguide and Zeplin component hovers

### CODE REVIEW

File: src/common/domain/openInZeplin/flow/openInZeplinFlow.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import * as vscode from "vscode";
```
Comment (fonurr) [inline src/common/domain/openInZeplin/flow/openInZeplinFlow.ts:1]: This is not a new file. It's relocated and a little bit changed. Previous version:
https://github.com/zeplin/vscode-extension/blob/3acb791eca73e7957c5ed51167b72e8ccbc55691/src/sidebar/openInZeplin/flow/openInZeplinFlow.ts

File: src/coco/zeplinComponent/hover/ZeplinComponentHoverCreator.ts [hunk 2]
```diff
@@ -24,24 +28,26 @@ class ZeplinComponentHoverCreator implements ConfigHoverCreator {
...
-            for (let componentIndex = 0; componentIndex < data!.length; componentIndex++) {
-                const { component, providerId, providerType } = data![componentIndex];
+            const allComponents = data!;
+            const componentsToDisplay = allComponents.slice(0, MAX_ELEMENTS_TO_DISPLAY);
+            const thumbnailSizes = await Promise.all(
+                componentsToDisplay.map(({ component }) => getImageSize(component.latestVersion.snapshot.url))
+            );
+            for (let componentIndex = 0; componentIndex < componentsToDisplay.length; componentIndex++) {
+                const componentData = componentsToDisplay[componentIndex];
+                const { component } = componentData;
```
Comment (mertkahyaoglu) [inline src/coco/zeplinComponent/hover/ZeplinComponentHoverCreator.ts:38]:
```suggestion
                const { component } = componentsToDisplay[componentIndex];
```

**Review by mertkahyaoglu — APPROVED** [review]
1 tiny comment.

File: src/coco/zeplinComponent/hover/ZeplinComponentHoverCreator.ts (hunk 2 above)
Comment (fonurr) [reply src/coco/zeplinComponent/hover/ZeplinComponentHoverCreator.ts:38]: componentData is also used below

**Review by mertkahyaoglu — APPROVED** [review]

**Review by konukselim — APPROVED** [review]

**Pull request merged**